### ISSUE TITLE
Ero

### ISSUE THREAD

**Issue by doberkofler** [stylelint/stylelint#2143]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

e.g. "A bug where..."

> Which rule, if any, is this issue related to?

e.g. `number-leading-zero`

> What CSS is needed to reproduce this issue?

e.g.

```css
.foo {
  color: pink;
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "indentation": ["tab", {
      "except": ["value"]
    }]
  }
}
```

> Which version of stylelint are you using?

e.g. `6.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "CLI with `stylelint --config myconfig *.css`"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

e.g. "Yes, it's related to SCSS maps."

> What did you expect to happen?

e.g. "No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
test.css
1:21    Expected 2 spaces (indentation)
```

**Comment by Arcanemagus** [comment]
@doberkofler You need to actually fill in the template above, currently your entire issue is only what you have in the title of "Ero", which isn't very helpful.

**Comment by doberkofler** [comment]
@Arcanemagus sorry. wrongly pressed enter but now have created a proper issue.
